src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -5958,4 +5958,438 @@ inline void compute_sum_24_host(__m256d *p, __m256d *pSumR, __m256d *pSumG, __m2
     pSumB[0] = _mm256_add_pd(_mm256_add_pd(p[4], p[5]), pSumB[0]); //add 8B values and bring it down to 4
 }
 
-#endif //RPP_CPU_COMMON_H
\ No newline at end of file
+inline void compute_min_32_host(__m256i *p1, __m256i *pMin)
+{
```

Comment:
please see prev comment

Reply:
please remove this inline function and use _mm256_min_epu8 directly

Reply:
Please check the branch after latest changes, I have removed this in commit "[Cleanup by removing oneliner functions as inline](https://github.com/ROCm/rpp/pull/260/commits/9a657c29991b0603ed9279f40e57824129866d65)"